README.md

```diff
@@ -104,3 +151,11 @@ In order to produce a uniquely identifiable distribution:
  * If the version of a package **is** being increased, please remember to return
    the [``build/number``](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#build-number-and-string)
    back to 0.
+
+Feedstock Maintainers
+=====================
+
+* [@SylvainCorlay](https://github.com/SylvainCorlay/)
+* [@dfornika](https://github.com/dfornika/)
```

Comment:
Could I be taken off this feedstock? I'm not sure how helpful I can be on this one.

Reply:
This is autogenerated from the maintainers list in the `meta.yaml`.